test/kyberNetwork.js

```diff
@@ -2469,7 +2470,11 @@ contract('KyberNetwork', function(accounts) {
                 //perform trade
                 let txData = await network.tradeWithHint(user, tokenAdd[tokenInd], amountTwei, ethAddress, user, 500000,
                                 rate[1].valueOf(), walletId, 0, {from:networkProxy});
-                console.log("Transaction gas used token to eth for " + (step + 1) + " steps: " + txData.receipt.gasUsed);
+                if (step < 2) {
+                    // only when step < 3, the number of steps for qty and imbalance are the same
+                    // when qty step is 4, or 5, the imbalance step is just 1 or 2
```

Comment:
I don't follow the comment.

"number of steps for qty and imbalance are..."
we don't have qty steps. what does this mean??

Reply:
Sr it's comment from previous on, updated